fluent-langneg/src/accepted_languages.js

```diff
@@ -1,7 +1,24 @@
-export default function acceptedLanguages(string = "") {
-  if (typeof string !== "string") {
+export default function acceptedLanguages(acceptLanguageHeader = "") {
+  if (typeof acceptLanguageHeader !== "string") {
     throw new TypeError("Argument must be a string");
   }
-  const tokens = string.split(",").map(t => t.trim());
-  return tokens.filter(t => t !== "").map(t => t.split(";")[0]);
+  const tokens = acceptLanguageHeader.split(",").map(t => t.trim());
+  const langsWithQ = [];
+  tokens.filter(t => t !== "").forEach((t, index) => {
+    const langWithQ = t.split(";").map(u => u.trim());
+    if (langWithQ[0].length > 0) {
```

Comment:
String.prototype.split always returns at least one element, so no need for this check I think?